### PR TITLE
Fix issue #7121 and two more bugs

### PR DESCRIPTION
**Pull request by woutervdub**

### CODE REVIEW

File: designs/octoplushy/src/head.mjs [hunk 1]
```diff
@@ -613,15 +613,16 @@ const options = {
     min: 5,
     max: 500,
     menu: 'style',
+    // eslint-disable-next-line no-unused-vars
```
Comment (HaasJona) [inline designs/octoplushy/src/head.mjs:616]: You can use `toAbs: (val, _, mergedOptions) => ` if you don't need the `options` variable and then don't need to disable the lint warning.

File: designs/octoplushy/src/head.mjs [hunk 2]
```diff
@@ -613,15 +613,16 @@ const options = {
...
     toAbs: (val, { options }, mergedOptions) =>
       ((mergedOptions.sizeConstant * val * 2) / 3.1415) *
```
Comment (HaasJona) [inline designs/octoplushy/src/head.mjs:618]: Unrelated to the fix, but could use `Math.PI` here for clarity

File: designs/octoplushy/src/head.mjs [hunk 3]
```diff
@@ -613,15 +613,16 @@ const options = {
...
-      (options.type === undefined
+      (mergedOptions.type === undefined
```
Comment (HaasJona) [inline designs/octoplushy/src/head.mjs:619]: Under which circumstances is the type undefined? Also wouldn't that be already handled by the `: 1` at the end?

File: designs/octoplushy/src/head.mjs [hunk 4]
```diff
@@ -613,15 +613,16 @@ const options = {
...
         ? 1
-        : options.type == 'octopus'
-        ? 1.7
-        : options.type == 'squid'
-        ? 2
-        : 1),
+        : mergedOptions.type == 'octopus'
```
Comment (HaasJona) [inline designs/octoplushy/src/head.mjs:621]: Is there a reason to not use `===` here?

**Review by BenJamesBen — APPROVED** [review]
I tested the PR in the lab, and it works as expected, with no issues seen.

File: designs/octoplushy/src/head.mjs (hunk 2 above)
Comment (woutervdub) [reply designs/octoplushy/src/head.mjs]: Done

File: designs/octoplushy/src/head.mjs (hunk 1 above)
Comment (woutervdub) [reply designs/octoplushy/src/head.mjs]: Done

File: designs/octoplushy/src/head.mjs (hunk 4 above)
Comment (woutervdub) [reply designs/octoplushy/src/head.mjs]: Don't think so. Removed

File: designs/octoplushy/src/head.mjs (hunk 3 above)
Comment (woutervdub) [reply designs/octoplushy/src/head.mjs]: Yes, that should not happen. Removed

File: designs/octoplushy/src/head.mjs (hunk 1 above)
Comment (HaasJona) [reply designs/octoplushy/src/head.mjs]: Use `_`, not `{}`, or you get another warning. 🙄

Comment (woutervdub) [reply designs/octoplushy/src/head.mjs]: I added an ignore line ;-)

File: designs/octoplushy/src/head.mjs [hunk 5]
```diff
@@ -613,15 +613,10 @@ const options = {
...
-      ((mergedOptions.sizeConstant * val * 2) / 3.1415) *
-      (options.type === undefined
-        ? 1
-        : options.type == 'octopus'
-        ? 1.7
-        : options.type == 'squid'
-        ? 2
-        : 1),
+    // eslint-disable-next-line no-empty-pattern
+    toAbs: (val, {}, mergedOptions) =>
```
Comment (joostdecock) [inline designs/octoplushy/src/head.mjs:617]: No need to destructure the second parameter. You can use `_` like on like 628 so that we're consistent.

**Review by joostdecock — CHANGES_REQUESTED** [review]
One small fix, then we're good to go.

**Review by joostdecock — APPROVED** [review]

**Pull request merged**